pyuvdata/uvcal/tests/test_fhd_cal.py

```diff
@@ -37,8 +38,12 @@ def test_read_fhdcal_write_read_calfits(tmp_path):
     fhd_cal.write_calfits(outfile, clobber=True)
     calfits_cal.read_calfits(outfile)
     assert fhd_cal == calfits_cal
+    del fhd_cal, calfits_cal
+    gc.collect()
```

Comment:
hmm a little nervous about additional explicit `gc` calls just to make things work again. But glad it is working. Maybe we should make an issue in case we want to come back to this and investigate.

Reply:
I think this is a vestige of a first-attempt that may not actually be needed. I'll reorganize and see if tests still pass.

Reply:
I've split tests up more now, so I took out the `gc` statements.